src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.revrobotics.SparkMaxLimitSwitch.Type;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.utils.MoPrefs;
+import frc.robot.utils.MotorMovement;
+
+public class ArmSubsystem extends SubsystemBase {
+    public static final Type SHOULDER_LIMIT = Type.kNormallyOpen;
+    public static final Type WRIST_LIMIT = Type.kNormallyOpen;
+
+    private final CANSparkMax leftShoulder = new CANSparkMax(
+        Constants.ARM_SHOULDER_LEFT.address, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private final CANSparkMax rightShoulder = new CANSparkMax(
+        Constants.ARM_SHOULDER_RIGHT.address, CANSparkMaxLowLevel.MotorType.kBrushless);
+    private final CANSparkMax wrist = new CANSparkMax(
+        Constants.ARM_WRIST.address, CANSparkMaxLowLevel.MotorType.kBrushless);
+
+    public ArmSubsystem() {
+        rightShoulder.setInverted(true); // TODO: Verify
+    }
+
+    @Override
+    public void periodic() {
+        if (MotorMovement.isOutOfBounds(leftShoulder, SHOULDER_LIMIT)) {
+            leftShoulder.stopMotor();
+        }
+        if (MotorMovement.isOutOfBounds(rightShoulder, SHOULDER_LIMIT)) {
+            rightShoulder.stopMotor();
+        }
+        if (MotorMovement.isOutOfBounds(wrist, WRIST_LIMIT)) {
+            wrist.stopMotor();
+        }
```

Comment:
I have two concerns here:

1. I don't like that there could be potentially a command telling the motor to go, while at the same time the subsystem periodic() is telling the motor to stop. Which the motor decides to do would depend on the execution order of commands vs. subsystem periodic. While we could in theory look at the scheduler documentation/source code, it feels like a footgun to rely on the implementation details of the scheduler. IMO, there should only ever be one place in the code that is setting the motors: maybe add a private `setMotor()` that does bounds checking, then call that method from `adjustArm()` and `adjustWrist()`.
In fact, I see that the `MotorMovement` helper already does this, which makes me question why you are doing it here as well?

3. I doubt `stopMotor()` will be enough to actually stop the arm/wrist from moving, since gravity will be acting to pull the mechanisms down. What we really need is velocity/position PID and trapezoidal motion planning. However, `stopMotor()` is probably good enough until we get that PID working.